src/App.scss

```diff
@@ -0,0 +1,38 @@
+.App {
```

Comment:
Since these styles are pretty specific to the `create-react-app` project, I agree with you that we should remove them.

To do so, do your changes in your code editor, create a commit with the corresponding message and push it to the current branch.
The PR will be updated automatically

Reply:
I've deleted the styles, however, I've maintained the files in case I need to write styles on them.

Reply:
great